README.md

```diff
@@ -40,6 +40,31 @@ Optionally you can pass a list of services to be rebuilt:
 ./bootstrap.sh up api web
 ```
 
+## Services pre-run setup
+Some services will need to perform a certain set of actions before they are able to start successfully. This might include eg. running migrations, setting up fixtures, etc.
+
+To accomodate this, this script will run all the setup scripts, every time they change since the last setup run (as per last git modification timestamp).
```

Comment:
I always feel weird reading docs written this way 😅 `this script will run all the setup scripts` well okay, but at this point in the text I have no clue how to set up them and how they should look like or if they already exist or not.

Reply:
> I prefere aproches like:

Some services will need to perform a certain set of actions before they are able to start successfully. This might include eg. running migrations, setting up fixtures, etc.

To accommodate this, create a `./setup` directory with `.sh` files you would like to run. Scripts are executed in alphabetical order as per bash. Scripts will run every time they change since the last setup run (as per last git modification timestamp).

Example setup scripts initialization
 ```bash
 mkdir -p ./setup
 touch ./setup/initialize-database.sh
 ```

(...)